lib/ddtrace/opentracer/tracer.rb

```diff
@@ -1,4 +1,4 @@
-# typed: true
+# typed: false
```

Comment:
```
lib/ddtrace/opentracer/tracer.rb:19: Passing a hash where the specific keys are unknown to a method taking keyword arguments https://srb.help/7019
    19 |        @datadog_tracer = Datadog::Tracer.new(**options)
                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

It's asking me to add all `Datadog::Tracer#initialize` arguments to this class, which I don't think is a good idea.
I couldn't find a way to disable Sorbet only for the `Datadog::OpenTracer::Tracer#initialize` method, so I had to disable it for the whole file 😢

Reply:
Yup. To be honest, I'm not sure Sorbet has been adding a lot of value since its initial addition (where it caught a few existing issues), so I guess once we add support for steep, it may make sense to touch base and decide what to do with Sorbet.

Reply:
It did find at least a couple issues for me during development since the introduction.

Reply:
Sorbet has cost me more time than it's saved, and it doesn't have a good means of ignoring things I know are okay, forcing me to ignore the whole file.

A discussion for another thread, but this is yet another exhibit for the case against it later.